### DISCUSSION TITLE
Does anybody have spare Sepolia ETH ?

### DISCUSSION THREAD

**Discussion by karthikeh02** [Cyfrin/foundry-full-course-cu#2109] [Q&A] [answered]
Most of the Sepolia Faucet demands 0.001 ETH in our mainnet (eg. Alchemy..) I have reached max limit in the chainlink faucet. Iam midway through the smart contract lottery lesson and iam insufficient of sepolia eth in my wallet . if anybody have unused eth please send it to me that would be helpful

wallet address : 0x329830b49B41b1f348420eECF79E1a73d8B45FB9

**Comment by EngrPips** [comment]
I hope someone is kind enough to help with this.

**Reply by EngrPips** [reply]
Someone told me they sent 0.2 ether to you. I hope that is enough.

**Reply by karthikeh02** [reply]
Yeah that is enough for now. @EngrPips i have a question, can you help me out?

**Reply by EngrPips** [reply]
Shoot your question.

**Reply by karthikeh02** [reply]
i have created a separate discussion for this question #2115

**Reply by EngrPips** [reply]
Alright.

**Comment by Moemen12** [comment, marked as answer]
i have sent some sepolia eth , if it's arrived dont forget to mark my answer as the correct answer ,thank u
[https://sepolia.etherscan.io/tx/0xabde69e46c6500bbf57f430db850e56e8b8690f9b093004e0dd6437cd69f9dcb](url)

**Reply by karthikeh02** [reply]
yeah i got your sepolia eth. @Moemen12 thanks for your help .

**Reply by Moemen12** [reply]
Can u mark my answer as the correct one plz

**Reply by karthikeh02** [reply]
sure thing

**Comment by Yuikamaa1** [comment]
> Most of the Sepolia Faucet demands 0.001 ETH in our mainnet (eg. Alchemy..) I have reached max limit in the chainlink faucet. Iam midway through the smart contract lottery lesson and iam insufficient of sepolia eth in my wallet . if anybody have unused eth please send it to me that would be helpful
> 
> wallet address : 0xDc72ac035b6977F9217566F41eA981BB238AaB55
or

Hi everyone,  
I need some Sepolia ETH for testing a smart contract.  
My wallet address: 0xDc72ac035b6977F9217566F41eA981BB238AaB55
Thank you in advance! 🙏

**Reply by EngrPips** [reply]
You can use this faucet https://cloud.google.com/application/web3/faucet/ethereum/sepolia or this faucet https://sepolia-faucet.pk910.de/

**Comment by khandelwalmoksh** [comment]
try this [https://chainlist.org/?testnets=true](https://t.co/RfBs0ypjoc) and Faucet Trade Bot (telegram) both recommended by Patrick Collins
